src/parkapi_sources/converters/apcoa/mapper.py

```diff
@@ -0,0 +1,83 @@
+
+from .validators import ApcoaNavigationLocationType, ApcoaOpeningHoursWeekdays, ApcoaParkingSiteInput, ApcoaParkingSpaceType
+
+
+class ApcoaMapper:
+    @staticmethod
+    def map_static_parking_site(apcoa_input: ApcoaParkingSiteInput) -> StaticParkingSiteInput:
+        static_parking_site_input = StaticParkingSiteInput(
+            uid=str(apcoa_input.CarParkId),
+            name=apcoa_input.CarparkLongName if apcoa_input.CarparkLongName else apcoa_input.CarparkShortName,
+            lat=next(
+                iter(
+                    location_type_input.GeoCoordinates.Latitude
+                    for location_type_input in apcoa_input.NavigationLocations
+                    if location_type_input.LocationType == ApcoaNavigationLocationType.DEFAULT
+                )
+            ),
+            lon=next(
+                iter(
+                    location_type_input.GeoCoordinates.Longitude
+                    for location_type_input in apcoa_input.NavigationLocations
+                    if location_type_input.LocationType == ApcoaNavigationLocationType.DEFAULT
+                )
+            ),
+            purpose=PurposeType.CAR,
+            type=apcoa_input.CarparkType.Name.to_parking_site_type_input(),
+            has_realtime_data=False,  # TODO: change this as soon as Apcoa API offers realtime data
+            static_data_updated_at=datetime.now(tz=timezone.utc),
+        )
+        if apcoa_input.CarParkWebsiteURL:
+            static_parking_site_input.public_url = apcoa_input.CarParkWebsiteURL
+
+        if apcoa_input.Address.Street and apcoa_input.Address.Zip and apcoa_input.Address.City:
+            static_parking_site_input.address = f'{apcoa_input.Address.Street}, {apcoa_input.Address.Zip} {apcoa_input.Address.City}'
+
+        num_of_weekdays = len(
+            [
+                opening_hours_input.Weekday
+                for opening_hours_input in apcoa_input.OpeningHours
+                if opening_hours_input.Weekday in ApcoaOpeningHoursWeekdays.Weekdays
+                and opening_hours_input.OpeningTimes == ApcoaOpeningHoursWeekdays.OpeningTime
+            ]
+        )
+        if num_of_weekdays == 7:
+            static_parking_site_input.opening_hours = '24/7'
+
+        for capacity_data in apcoa_input.Spaces:
+            # Because it was checked in validation, we can be sure that capacity will be set
+            if capacity_data.Type == ApcoaParkingSpaceType.TOTAL_SPACES:
+                static_parking_site_input.capacity = int(round(capacity_data.Count))
+            elif capacity_data.Type == ApcoaParkingSpaceType.DISABLED_SPACES:
+                static_parking_site_input.capacity_disabled = int(round(capacity_data.Count))
+            elif capacity_data.Type == ApcoaParkingSpaceType.WOMEN_SPACES:
+                static_parking_site_input.capacity_woman = int(round(capacity_data.Count))
+            elif capacity_data.Type == ApcoaParkingSpaceType.ELECTRIC_CAR_CHARGING_SPACES:
```

Comment:
Why not using `capacity_data.Type in [ApcoaParkingSpaceType.ENUM_1, ApcoaParkingSpaceType.ENUM2]`?